client.lua

```diff
@@ -182,83 +170,73 @@ local function grapeJuiceProcess()
 	ClearPedTasks(PlayerPedId())
 end
 
-local function insideStartZone()
+local function insideStartZone(self)
```

Comment:
Consider renaming to a more descriptive function name

Reply:
This should be better now.